packages/clay-core/src/table/Cell.tsx

```diff
@@ -110,31 +126,40 @@ export const Column = React.forwardRef<HTMLTableCellElement, Props>(
 					'table-cell-ws-nowrap': !wrap,
 					'table-head-title': isHead,
 				})}
+				data-id={
+					typeof keyValue === 'number'
+						? `number,${keyValue}`
+						: `string,${keyValue}`
+				}
+				onClick={(event) => {
+					if (!(isHead && sortable)) {
+						return;
+					}
+
+					event.preventDefault();
```

Comment:
In portal I understand Brian's rule for when to add a blank line between items: If you can't sort the items add a line between them. But, what's our rule in Clay for when to add a blank line?

Reply:
@ethib137 we don't adopt the spacing rule so much here, we just use something for visuals that is clear to understand when reading, we prioritize that.